### PR TITLE
Update ElasticSearch to 2.2.1

### PR DESCRIPTION
**Pull request by chibenwa**

### CODE REVIEW

**Comment by aduprat** [comment]
Not part of this issue, but has you are working on this code, we should also:
- give a version to Cassandra
- give a name to Cassandra & ES containers (to avoid name collision)

**Comment by chibenwa** [comment]
Ok, sure...

**Comment by chibenwa** [comment]
I gave a version to Cassandra : 2.1.9 (based cassandra-unit version)

For the naming, no need as the up to date workflowjob contains : 

```
sh "docker run --detach=true --name=${containers.cassandra} ${images.cassandra}"
sh "docker run --detach=true --name=${containers.elasticSearch} ${images.elasticSearch}"
```

with 

```
public class Containers implements Serializable {

    def cassandra;
    def elasticSearch;

    public Containers(String buildId) {
        cassandra = 'cassandra-' + buildId;
        elasticSearch = 'elasticSearch-' + buildId;
    }
}
```

As these images are external (and thus, once the version specified immutable) no need to give them a name.

File: workflow-job [hunk 1]
```diff
@@ -95,7 +95,7 @@ public class CommitStatus implements Serializable {
 }
 
 public class Images implements Serializable {
-    def cassandra = 'cassandra'
+    def cassandra = 'cassandra:2.1.9'
```
Comment (aduprat) [inline workflow-job]: I think we have to stay consistent, in the James documentation (README in james-project) we are giving the command with the `2.2.3` release.

Comment (chibenwa) [inline workflow-job]: Updated.
If we say we support cassandra 2.2.3 we should use version 2.2.2.1 of cassandra unit and not 2.1.9.2

See https://github.com/jsevellec/cassandra-unit/wiki/changelog

Do you also want me to contribute this update in James sources ?

Comment (aduprat) [inline workflow-job]: Good idea!

Comment (chibenwa) [inline workflow-job]: When updating Cassandra unit, I get these strange errors : 

```
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] no libsigar-amd64-linux.so in java.library.path
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18] org.hyperic.sigar.SigarException: no libsigar-amd64-linux.so in java.library.path
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18]  at org.hyperic.sigar.Sigar.loadLibrary(Sigar.java:172)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18]  at org.hyperic.sigar.Sigar.<clinit>(Sigar.java:100)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18]  at org.apache.cassandra.utils.SigarLibrary.<init>(SigarLibrary.java:45)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18]  at org.apache.cassandra.service.StartupChecks$5.execute(StartupChecks.java:178)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18]  at org.apache.cassandra.service.StartupChecks.verify(StartupChecks.java:103)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18]  at org.apache.cassandra.service.CassandraDaemon.setup(CassandraDaemon.java:162)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18]  at org.apache.cassandra.service.CassandraDaemon.activate(CassandraDaemon.java:511)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18]  at org.cassandraunit.utils.EmbeddedCassandraServerHelper$1.run(EmbeddedCassandraServerHelper.java:123)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18]  at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18]  at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[Building Java 8 mailbox-266-f96d54c7-aa6e-4a79-ae35-bc3efa7cbe18]  at java.lang.Thread.run(Thread.java:745)
```

As it is not directly linked to MAILBOX-266 ticket, I reset my branch and will not investigate on this much...

**Comment by aduprat** [comment]
:+1:

**Comment by rouazana** [comment]
:+1:

**Pull request merged**